readthedocs/profiles/views.py

```diff
@@ -279,3 +280,34 @@ def profile_detail(
 
     context.update({'profile': profile_obj})
     return render(request, template_name, context=context)
+
+
+def account_advertising(request):
```

Comment:
Does this need a login_requires decorator? Guessing it will blow up with a non-logged in user otherwise?